tests/SMC.test_binance.py

```diff
@@ -308,12 +314,12 @@ def add_liquidity(fig):
     return fig
 
 
-fig = add_FVG(fig)
-fig = add_highs_lows(fig)
-fig = add_bos_choch(fig)
-fig = add_OB(fig)
+# fig = add_FVG(fig)
+# fig = add_highs_lows(fig)
+# fig = add_bos_choch(fig)
+# fig = add_OB(fig)
 fig = add_volumized_OB(fig)
-fig = add_liquidity(fig)
+# fig = add_liquidity(fig)
```

Comment:
Please keep all of the test functions uncommented on the final commit

Reply:
ok